sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1361,30 +1361,35 @@ public void Cfg_Coalesce()
 
             VerifyAllInstructions(branchBlock, "b");
```

Comment:
now that you touch this code, could you please replace `branchBlock.SuccessorBlocks.Should().OnlyContainInOrder(bNullBlock, afterOp)` with `TrueSuccessor.Should()...` and `FalseSuccessor.Should()...`)?